docs/docs/kb/09-docker.md

```diff
@@ -31,18 +31,27 @@ docker run -ti ignitehq/cli chain serve
 
 When Docker is installed, you can build a blockchain with a single command.
 
+Ignite CLI, and the chains you serve with Ignite CLI, persist some files.
+When using the CLI binary directly, those files are located in `$HOME/.ignite`
+and `$HOME/.cache`, but in the context of Docker it's better to use a directory different than `$HOME`, so we use `$HOME/sdh`. This folder should be created
+manually prior to the docker commands below, or else Docker creates it with the
+root user.
+
+```bash
+mkdir $HOME/sdh
+```
+
 To scaffold a blockchain `planet` in the `/apps` directory in the container, run this command in a terminal window:
 
 ```bash
-docker run -ti -w /app -v $HOME/sdh:/home/tendermint -v $PWD:/app ignitehq/cli:0.16.0 app github.com/hello/planet
+docker run -ti -v $HOME/sdh:/home/tendermint -v $PWD:/apps ignitehq/cli:0.16.0 scaffold chain github.com/hello/planet
 ```
 
 Be patient, this command takes a minute or two to run because it does everything for you:
 
 - Creates a container that runs from the `ignitehq/cli:0.16.0` image.
 - Executes the Ignite CLI binary inside the image.
-- `-w /apps` sets the current directory in the container to `/app`
-- `-v $HOME/sdh:/home/tendermint` maps the `$HOME/sdh` directory in your local computer (the host machine) to the home directory `/home/tendermint` inside the container. Ignite CLI, and the chains you serve with Ignite CLI, persist some files.
+- `-v $HOME/sdh:/home/tendermint` maps the `$HOME/sdh` directory in your local computer (the host machine) to the home directory `/home/tendermint` inside the container. 
 - `-v $PWD:/app` maps the current directory in the terminal window on the host machine to the `/app` directory in the container. You can optionally specify an absolute path instead of `$PWD`.
```

Comment:
```suggestion
- `-v $PWD:/apps` maps the current directory in the terminal window on the host machine to the `/apps` directory in the container. You can optionally specify an absolute path instead of `$PWD`.
```

Reply:
oh thks I missed that one.